src/main/kotlin/racingcar/RacincarApp.kt

```diff
@@ -0,0 +1,9 @@
+import racingcar.InputView
+import racingcar.RacingGame
+
+fun main() {
+    val carCnt = InputView("자동차 대수는 몇 대인가요?").value
+    val tryCnt = InputView("시도할 횟수는 몇 회인가요?").value
```

Comment:
```suggestion
    val inputView = InputView()
    val carCnt = inputView.readCarCount()
    val tryCnt = inputView.readTryCount()
```

위의 방식으로 바꾸는것도 고민해볼까요?
`.value`를 하면 변수를 읽는것 같지만, 실제로는 콘솔에서 데이터를 읽고 있어요.
`.value`를 자바 코드로 변경하면 `.getValue()`와 같은데, 이런 코드는 단순하게 변수를 가져오는것을 의미합니다.

`InputView`에게 너 ~~좀 해줘! 라고 명령하면(함수 호출), 다른 개발자도 이해하기 편할거에요.

다른 개발자가 보았을때도 명확한 코드면 좋겠죠?